Brewfile

```diff
@@ -4,4 +4,5 @@ brew 'redis'
 brew 'node@16'
```

Comment:
I think this is out of date now

Reply:
oh interesting! What do you think it should be?

Reply:
[18](https://github.com/18F/identity-idp/blob/main/.nvmrc#L1) for now (and eventually [20](https://github.com/18F/identity-idp/pull/9639))

Reply:
not sure honestly, don't necessarily have to address it in this PR though

Reply:
I can test out 18 quick and put up a PR for moving to 18 if it's successful after merging this.